charts/zalenium/templates/_pod-template.yaml

```diff
@@ -9,6 +9,10 @@ spec:
     fsGroup: {{ .Values.hub.securityContext.fsGroup }}
     runAsUser: {{ .Values.hub.securityContext.runAsUser }}
   {{- end }}
+  {{- if .Values.hub.pullSecrets }}
+  imagePullSecrets:
+    - name: {{ .Values.hub.pullSecrets }}
```

Comment:
Please rename `pullSecrets` to `imagePullSecret` to follow kubernetes pod template semantic and as you set only one secret.